spec/models/profile_spec.rb

```diff
@@ -66,6 +63,27 @@
       expect(profile.initiating_service_provider).to be_nil
       expect(profile.verified_at).to be_nil
     end
+
+    context 'with unlinked establishing enrollment' do
+      # When marking profile as pending in-person proofing, the enrollment hasn't yet been linked to
+      # the profile.
+      #
+      # See: Idv::Session#create_profile_from_applicant_with_password
+
+      before do
```

Comment:
style, I'd remove the space between?
```suggestion
      # See: Idv::Session#create_profile_from_applicant_with_password
      before do
```

Reply:
Hm, yeah. The padding line was intentional to try to "link" it more with the `context` block than the `before` block. But maybe in that case the comment should live above the `context` line? Not sure how I feel about this.

Reply:
I guess my thinking is the context description is what we want to happen, the the before block (or sometimes a let) is **how** we want that to happen, and in this case, the comment clarifies the how